docs/CHANGELOG.txt

```diff
@@ -23,6 +23,54 @@ needs please refer to https://devdocs.prestashop.com/ for more information.
 
 Release Notes for PrestaShop 1.7
 
+####################################
+#   v1.7.8.3 - (2022-01-XX)
```

Comment:
```suggestion
#   v1.7.8.3 - (2022-01-06)
```

Reply:
@atomiix You're sure ?
![image](https://user-images.githubusercontent.com/16455155/148522297-64984a04-36c6-4860-ac24-f70cbdb30b9c.png)
https://github.com/PrestaShop/PrestaShop/issues/26969

Reply:
This date is the build date, not the release date! But I'm still wrong, it was built on the 5th 😅 I'll update my suggestion

Reply:
I agree with @atomiix